### PR TITLE
🚀 new possibility to escape special routing parameters

### PR DESCRIPTION
**Pull request by ReneWerner87**
🚀 new possibility to escape special routing parameters,
which gives the possibility to follow the google api design guide https://cloud.google.com/apis/design/custom_methods

### CODE REVIEW

**Comment by mlukasik-dev** [comment]
Oh, that's really cool! Thank you very much this feature will be very helpful :)

File: path.go [hunk 1]
```diff
@@ -339,3 +360,11 @@ func GetTrimmedParam(param string) string {
 
 	return param[start:end]
 }
+
+// RemoveEscapeChar remove escape charaters
+func RemoveEscapeChar(word string) string {
+	if -1 != strings.IndexByte(word, escapeChar) {
```
Comment (mlukasik-dev) [inline path.go:366]: @ReneWerner87 , just asking: don't you think that `strings.IndexByte(word, escapeChar) != -1` is a bit clearer and in sync with "the best practices"?)

**Comment by ReneWerner87** [comment]
gofiber/docs#165

File: path.go (hunk 1 above)
Comment (ReneWerner87) [reply path.go:366]: Yes, I can do that too, I am used to the yoda notation from other programming languages, should also bring only advantages here

**Pull request merged**